language/Portuguese_Brazil/strings.po

```diff
@@ -416,30 +416,30 @@ msgstr ""
 
 #: src/frontend/mame/ui/info.cpp:176
 msgid "Completely unemulated features: "
-msgstr ""
+msgstr "Características completamente não emuladas"
 
 #: src/frontend/mame/ui/info.cpp:182 src/frontend/mame/ui/info.cpp:198
 #, c-format
 msgid "%s"
-msgstr ""
+msgstr "%s"
 
 #: src/frontend/mame/ui/info.cpp:182 src/frontend/mame/ui/info.cpp:198
 #: src/frontend/mame/ui/info.cpp:243
 #, c-format
 msgid ", %s"
-msgstr ""
+msgstr "%s"
 
 #: src/frontend/mame/ui/info.cpp:192
 msgid "Imperfectly emulated features: "
-msgstr ""
+msgstr "Características emuladas imperfeitamente"
```

Comment:
This introduces a list, hence the colon/space after it. If you don't have space/appropriate punctuation you'll end up with:
```
Características emuladas imperfeitamentedisco
```
Note lack of space turning two words into one.